scripts/solve_network.py

```diff
@@ -60,18 +63,18 @@ def _add_land_use_constraint(n):
         existing.index += " " + carrier + "-" + snakemake.wildcards.planning_horizons
         n.generators.loc[existing.index, "p_nom_max"] -= existing
 
-    # check if existing capacities are larger than technical potential
```

Comment:
I am not sure if it is good to remove this part. I guess it can also happen that the existing capacities which are non-extendable are already larger than the technical potential which leads to infeasibilities. So it would be nice to keep the warning and the following part.

Reply:
I think you are right, this is very unlikely but not impossible, so maybe we just keep it.

Reply:
I think I already got this warning especially with a higher spatial resolution, so I would favor to keep it

Reply:
I think I already got this warning especially with a higher spatial resolution, so I would favor to keep it since otherwise the network is infeasible to solve

Reply:
we agree, I just reverted the commented lines